### DISCUSSION TITLE
Change yourls settings

### DISCUSSION THREAD

**Discussion by developer20ec** [YOURLS/YOURLS#3301] [Q&A]
Hi everyone.
I have a webapp with the integrated YOURLS service that allows me to generate short url in 2 different ways:
- in "basic" version with 4 random numerical parameters "... /4567"
- in "customized" version that replaces the first number with the username of the webapp user "... /name567"

The activated plugins are:
"Allow Hyphens in Short URLs"
"Custom API action"
"Domain Limiter"

I would like to know, if possible:
1. When a short url is deleted (for example "... / 1234") is it "burned" or can it be reused?
2. When using the custom version, does the username replace a number? If so, which one?
3. What are the reference files for these settings?

Thanks in advance for your answers.

**Comment by ozh** [comment]
Hello

Out of the box, when a short URL is deleted, it can be reused if you pick it (ie you create a short URL and input the keyword). If no keyword is given when the short URL is created, it won't be reused.

Questions 2 and 3 are irrelevant to YOURLS, this depends how YOURLS was integrated in your webapp. The "basic" and "customized" methods you describe either depend on a plugin, or on your webapp.
